src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header_logo">
+        <a href="https://www.moyo.ua/">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header_section">
+        <ul class="header_nav">
+          <li class="header_nav_item">
+            <a class="link_item is-active" href="http://127.0.0.1:5500/src/index.html">Apple</a>
```

Comment:
```suggestion
            <a class="link_item is-active" href="#Apple">Apple</a>
```

Fix everywhere to prevent page reloading